Tests/ScipioKitTests/RunnerTests.swift

```diff
@@ -566,7 +566,12 @@ final class RunnerTests: XCTestCase {
         let runner = Runner(
             mode: .prepareDependencies,
             options: .init(
-                baseBuildOptions: .init(isSimulatorSupported: true),
+                baseBuildOptions: .init(
+                    isSimulatorSupported: true,
+                    extraBuildParameters: [
+                        "EXCLUDED_ARCHS": "i386",
```

Comment:
Xcode 16.3 drops i386 support. We don't need to check i386 support so I disabled it.